### ISSUE TITLE
How can I speed up the training process?

### ISSUE THREAD

**Issue by pumfish** [PhilipZRH/LaNE#1]
Hello, thank you very much for open-sourcing such an interesting project. 
I am conducting experiments using your project, and I noticed that when I use the default script parameters, my GPU's memory and computational resources are only utilizing a small portion. What parameters can I modify to speed up my training process? 
When I increase parameters such as ```frame_stack``` and ```batch_size```, can it speed up the process?
Additionally, I would like to ask if it is normal for the reward to reach 100 and various losses to be 0 during the initial stage of training.
![image](https://github.com/user-attachments/assets/7c378818-a818-45c9-8c73-b00a496cb607)

**Comment by PhilipZRH** [comment]
Hello, thanks for your interest in this project!

- A bigger batch size could help the world model and SAC policy converge more quickly.
- Additionally, you can tweak the E2C model update schedules around line 627 in `sac.py`
- `frame_stack` is not necessarily related to the training convergence speed. It mostly alleviates partial observability issues by combining a few historical observations into one.

For the reward issue:
- Please checkout line 153 in `env_wrapper.py`, where we defined the sparse reward function.
- From your logs, it seems that all episodes are considered successful immediately after reset.

I hope this answers your questions!

**Comment by pumfish** [comment]
Thank you very much for your response. I forgot that I had previously modified the Robosuite environment. I have resolved this bug, and now the stack environment is running smoothly.

Additionally, as I am trying to replicate the performance of Lane on Stack, I have increased the batch size to 256 while keeping other settings consistent with the source code. However, the success rate of my stack task remains at 0. Could you please advise if there are any specific considerations to keep in mind when training a stack task?
![image](https://github.com/user-attachments/assets/2d209c89-48ce-4adc-bc96-eeff4b0d4be5)

**Comment by PhilipZRH** [comment]
Looking at your screenshot where avg_discount is NaN, I suspect there’s issue during E2C module updates causing the embedding to become NaNs. Maybe the change in batch size has made the gradients explode. Could you try troubleshoot along that direction, say decrease the E2C learning rate?

**Comment by pumfish** [comment]
Thanks for your suggestions. I will try to debug this issue following this approach
